randovania/games/am2r/logic_database/Industrial Complex.txt

```diff
@@ -473,8 +473,9 @@ Extra - light_level: 0
       All of the following:
           Any of the following:
               # Weapon Requirements
-              Missiles ≥ 50 or Spazer Beam or Super Missiles ≥ 6 or Wave Beam or Combat (Intermediate)
+              Missiles ≥ 50 or Spazer Beam or Super Missiles ≥ 6 or Wave Beam or Combat (Advanced)
               Power Bombs ≥ 4 and Can Use Power Bombs
+              Long Beam and Combat (Intermediate)
```

Comment:
so on intermediate you are expected to fight Torizo with just long beam for weapons? Seems difficult
unless this is supposed to go in the `all`

Reply:
see genesis comment.